index.html

```diff
@@ -15,99 +15,113 @@
 <!-- Navigation -->
 
 
-<header>
-	<h1>S&J</h1>
-	<nav>
-		<a>Home</a>
-		<a>Services</a>
-		<a>Contact</a>
-	</nav>
-</header>
-
-
-
+<div class="header-container">
+	<div class="nav">
+		<h1>S&J</h1>
+		<img src="img/nav-hamburger.png" alt="Hamburger nav">
+		<nav class="nav-items">
+			<a>Home</a>
+			<a>Services</a>
+			<a>Contact</a>
+		</nav>
+	</div>
+</div>
```

Comment:
I would suggest that you add the routes for the pages as you  set up the anchor tags.